ops/services/app_functions/report_stream_batched_publisher/infra/_data.tf

```diff
@@ -15,6 +15,11 @@ data "azurerm_key_vault_secret" "datahub_api_key" {
   key_vault_id = data.azurerm_key_vault.sr_global.id
 }
 
+data "azurerm_key_vault_secret" "report_stream_callback_token" {
```

Comment:
how hard is it to rename this after the fact? It should be named `simple_report_callback_token` since it for reporting RS warnings/errors into the simple report db